go/internal/application/usecase/implementation/create_post.go

```diff
@@ -25,7 +27,18 @@ func (u *createPostUsecase) CreatePost(userID, text string) (entity.TimelineItem
 		return post, usecase.ErrTooLongText
 	}
 
-	post, err := u.timelineItemsRepository.CreatePost(userID, text)
+	users, err := u.usersRepository.FollowersByID(nil, userID)
+	var userIDs []string
+	if err != nil {
+		return post, err
+	}
```

Comment:
It's better to set the capacity up front when the length is known.

```suggestion
	if err != nil {
		return post, err
	}
	
    userIDs := make([]string, 0, len(users)+1)
```